src/ui/localization/locale/en.json

```diff
@@ -0,0 +1,57 @@
+{
+  "periods.week": "{{amount}} week",
+  "periods.month": "{{amount}} month",
+  "periods.year": "{{amount}} year",
+  "periods.day": "{{amount}} day",
+  "periods.weekShort": "{{amount}}wk",
+  "periods.monthShort": "{{amount}}mo",
+  "periods.yearShort": "{{amount}}yr",
+  "periods.dayShort": "{{amount}}d",
+  "periods.lifetime": "lifetime",
+  "periods.weekPlural": "{{amount}} weeks",
+  "periods.monthPlural": "{{amount}} months",
+  "periods.yearPlural": "{{amount}} years",
+  "periods.dayPlural": "{{amount}} days",
+  "periods.weekFrequency": "weekly",
+  "periods.monthFrequency": "monthly",
+  "periods.yearFrequency": "yearly",
+  "periods.dayFrequency": "daily",
+  "periods.unknownFrequency": "unknown",
+  "periods.weekFrequencyPlural": "every {{amount}} weeks",
+  "periods.monthFrequencyPlural": "every {{amount}} months",
+  "periods.yearFrequencyPlural": "every {{amount}} years",
+  "periods.dayFrequencyPlural": "every {{amount}} days",
+  "state_present_offer.product_title": "{{productTitle}}",
+  "state_present_offer.product_description": "{{productDescription}}",
+  "state_present_offer.product_price": "{{productPrice}}",
+  "state_present_offer.free_trial_duration": "{{trialDuration}} free trial",
+  "state_present_offer.price_after_free_trial": "{{productPrice}} after end of trial",
+  "state_present_offer.renewal_frequency": "Renews {{frequency}}",
+  "state_present_offer.continues_until_cancelled": "Continues until cancelled",
+  "state_present_offer.cancel_anytime": "Cancel anytime",
```

Comment:
This feels kind of repetitive, doesn’t it? I'd suggest discarding `state_present_offer.continues_until_cancelled`

Reply:
The specific labels were repeated to allow developers to customize whatever was there.
That's why all the labels are there even if they say the same thing in our current choices, they might be customized to say something else by the customers.

Reusing the same label might cause one customization in one point to affect other ones.

Reply:
But we're always showing these 3 lines, right?

Reply:
I think I am not understanding your comment. 

What do you mean by repetitive?
Do you want to remove the actual lines from the component or do you want to reuse 2 times the same label?